src/powershellScripts.ts

```diff
@@ -79,12 +85,17 @@ function fold ($children, $Block) {
     $hashset.Clear() | Out-Null
 }
 
-$found = Discover-Test -Path $Path
-
-# whole suite
-$suite = [PSCustomObject]@{
-    Blocks = [Collections.Generic.List[Object]] $found
-    Tests = [Collections.Generic.List[Object]]@()
+if (($PesterVersion.Major -eq 5 -and $PesterVersion.Minor -ge 2) -or
```

Comment:
Version is comparable, you can simply do `if ($PesterVersion -ge '5.2.0') {Do-NewLogic} else {Do-OldLogic`